### PR TITLE
feat(slider): add cta connotation (VIV-1373)

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: libs/components/src/lib/slider/slider.spec.ts [hunk 1]
```diff
@@ -139,6 +140,31 @@ describe('vwc-slider', () => {
 		});
 	});
 
+	describe('connotation', function () {
+		const possibleConnotations = [Connotation.Accent,
+			Connotation.CTA
+		];
```
Comment (RichardHelm) [inline libs/components/src/lib/slider/slider.spec.ts:146]:
```suggestion
		] as const;
```

File: libs/components/src/lib/slider/slider.spec.ts [hunk 2]
```diff
@@ -139,6 +140,31 @@ describe('vwc-slider', () => {
...
+
+		it('should leave connotation class empty if not set', async function () {
+			possibleConnotations.forEach(connotation => {
+				expect(getControlElement(element)
+					?.classList
+					.contains(connotation))
+					.toEqual(false);
+			});
+		});
```
Comment (RichardHelm) [inline libs/components/src/lib/slider/slider.spec.ts:155]: Not sure if this test is really needed
```suggestion
		it('should not set any connotation classes when no connotation is set', async function () {
			possibleConnotations.forEach(connotation => {
				expect(getControlElement(element)
					?.classList
					.contains(`connotation-${connotation}`))
					.toEqual(false);
			});
		});
```

File: libs/components/src/lib/slider/slider.spec.ts [hunk 3]
```diff
@@ -139,6 +140,31 @@ describe('vwc-slider', () => {
...
+		it('should set a connotation class', async function () {
+			const connotation = possibleConnotations[1];
+			(element.connotation as Connotation) = connotation;
+			await elementUpdated(element);
+			expect(getControlElement(element)
+				?.classList
+				.contains(`connotation-${connotation}`))
+				.toEqual(true);
+		});
+	});
```
Comment (RichardHelm) [inline libs/components/src/lib/slider/slider.spec.ts:166]: I think we can write it like this for all connotations
```suggestion
		it.each(possibleConnotations)('should set the connotation class for "%s"', async function (connotation) {
			element.connotation = connotation;
			await elementUpdated(element);
			expect(getControlElement(element)
				?.classList
				.contains(`connotation-${connotation}`))
				.toEqual(true);
		});
	});
```

**Review by RichardHelm — CHANGES_REQUESTED** [review]
Nice 👍  Got some minor comments

**Review by rachelbt — COMMENTED** [review]
LGTM,
the only thing is that is missing is the export of the connotation in the definition.ts:
export type { SliderConnotation } from './slider';

**Review by RichardHelm — APPROVED** [review]

**Pull request merged**